lib/ble.toit

```diff
@@ -393,10 +462,18 @@ class RemoteDevice extends Resource_:
 
   /**
   Disconnects from the remote device.
+
+  If $force is true, the connection is closed immediately. Otherwise, the
+    method waits until the remote device has acknowledged the disconnection.
   */
-  close:
+  close --force/bool=false:
+    services := discovered-services
```

Comment:
It feels a bit icky that the field is public.

Reply:
done.